etc/ReplayOfflineConfiguration.py

```diff
@@ -148,9 +148,7 @@
 # Setup repack and express mappings
 #
 repackVersionOverride = {
-    "CMSSW_12_6_1" : defaultCMSSWVersion['default'],
-    "CMSSW_12_6_2" : defaultCMSSWVersion['default'],
-    "CMSSW_12_6_3" : defaultCMSSWVersion['default']
+    "CMSSW_12_4_9_patch1" : "CMSSW_12_4_9_patch1"
```

Comment:
This line will work, but is not necessary. We will use the streamers version by default.

Reply:
ah ok! thanks for the explanation!